.factorypath

```diff
@@ -0,0 +1,16 @@
+<factorypath>
+    <factorypathentry kind="VARJAR" id="M2_REPO/org/slf4j/slf4j-api/1.7.25/slf4j-api-1.7.25.jar" enabled="true" runInBatchMode="false"/>
```

Comment:
Should this be added to .gitignore and not checked in ? Isn't it a generated file?

Reply:
Right. sorry I should have checked. will remove.